__tests__/app.test.js

```diff
@@ -15,3 +26,33 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe("GET /api/topics", () => {
+  test("200: responds with a list of all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        const { topics } = body;
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
+      });
+  });
+
+  test("404: responds with 404 error on wrong endpoint", () => {
+    return request(app)
+      .get("/api/topic")
```

Comment:
This test is fine, but it's not specific to this endpoint. For example, someone could write /api/banana and it would also be picked up by this 404. It should therefore be put into it's own describe block for 404 errors for incorrect endpoints 👍